### ISSUE TITLE
kublet.service fails to run kube pods

### ISSUE THREAD

**Issue by daveey** [coreos/coreos-kubernetes#440]
I have followed the kube-aws instructions to set up a cluster in us-east-1. The cloudformation stack gets created, but the controller fails to run the kube-apiserver. 

The journalctl logs from the controller, after starting the kublet.service are:

```
Apr 27 20:19:35 ip-10-0-0-50.ec2.internal polkitd[777]: Registered Authentication Agent for unix-process:2568:7292777 (system bus name :1.59 [/usr/bin/pkttyagent --notify-fd 4 --fallback], object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8)
Apr 27 20:19:35 ip-10-0-0-50.ec2.internal systemd[1]: Started kubelet.service.
Apr 27 20:19:35 ip-10-0-0-50.ec2.internal polkitd[777]: Unregistered Authentication Agent for unix-process:2568:7292777 (system bus name :1.59, object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8) (disconnected from bus)
Apr 27 20:19:35 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: image: using image from file /usr/share/rkt/stage1-fly.aci
Apr 27 20:19:36 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: image: using image from local store for image name quay.io/coreos/hyperkube:v1.2.2_coreos.0
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:42.322921    2573 server.go:445] Could not load kubeconfig file /var/lib/kubelet/kubeconfig: stat /var/lib/kubelet/kubeconfig: no such file or directory. Trying auth path instead.
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:42.323355    2573 server.go:406] Could not load kubernetes auth path /var/lib/kubelet/kubernetes_auth: stat /var/lib/kubelet/kubernetes_auth: no such file or directory. Continuing with defaults.
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.323636    2573 plugins.go:71] No cloud provider specified.
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.323904    2573 manager.go:132] cAdvisor running in container: "/system.slice/kubelet.service"
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.559271    2573 fs.go:109] Filesystem partitions: map[overlay:{mountpoint:/ major:0 minor:101 fsType:overlay blockSize:0} /dev/xvda9:{mountpoint:/var/lib/docker major:202 minor:9 fsType:ext4 blockSize:0} /dev/xvda3:{mountpoint:/etc/ssl/certs major:202 minor:3 fsType: blockSize:0}]
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.567423    2573 manager.go:169] Machine: {NumCores:1 CpuFrequency:2494074 MemoryCapacity:3950419968 MachineID:c0bb99276bc74b86b88d83396764e3f7 SystemUUID:EC2EBC46-10BD-1516-1955-817E485C66E1 BootID:e36b8501-aec4-4d63-8761-2368a05e5dc8 Filesystems:[{Device:overlay Capacity:28730269696 Type:vfs Inodes:7231872} {Device:/dev/xvda9 Capacity:28730269696 Type:vfs Inodes:7231872} {Device:/dev/xvda3 Capacity:1031946240 Type:vfs Inodes:260096}] DiskMap:map[202:0:{Name:xvda Major:202 Minor:0 Size:32212254720 Scheduler:none} 202:16:{Name:xvdb Major:202 Minor:16 Size:4289200128 Scheduler:none}] NetworkDevices:[{Name:eth0 MacAddress:0e:3a:74:ef:6a:e9 Speed:0 Mtu:9001} {Name:flannel.1 MacAddress:2e:62:60:fc:5e:51 Speed:0 Mtu:8951}] Topology:[{Id:0 Memory:3950419968 Cores:[{Id:0 Threads:[0] Caches:[{Size:32768 Type:Data Level:1} {Size:32768 Type:Instruction Level:1} {Size:262144 Type:Unified Level:2}]}] Caches:[{Size:26214400 Type:Unified Level:3}]}] CloudProvider:AWS InstanceType:m3.medium InstanceID:i-273c36bc}
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.568960    2573 manager.go:175] Version: {KernelVersion:4.5.2-coreos ContainerOsVersion:CoreOS 1029.0.0 (MoreOS) DockerVersion:1.10.3 CadvisorVersion: CadvisorRevision:}
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.571232    2573 server.go:673] Adding manifest file: /etc/kubernetes/manifests
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.571476    2573 server.go:683] Watching apiserver
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:42.589333    2573 kubelet.go:508] Hairpin mode set to "promiscuous-bridge" but configureCBR0 is false, falling back to "hairpin-veth"
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.590074    2573 kubelet.go:380] Hairpin mode set to "hairpin-veth"
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.815434    2573 manager.go:212] Setting dockerRoot to /var/lib/docker
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.816702    2573 plugins.go:56] Registering credential provider: .dockercfg
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.817275    2573 server.go:645] Started kubelet
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:42.817380    2573 kubelet.go:956] Image garbage collection failed: unable to find data for container /
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.817425    2573 server.go:109] Starting to listen on 0.0.0.0:10250
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.832499    2573 fs_resource_analyzer.go:66] Starting FS ResourceAnalyzer
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.832752    2573 manager.go:123] Starting to sync pod status with apiserver
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.832971    2573 kubelet.go:2356] Starting kubelet main sync loop.
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:42.833177    2573 kubelet.go:2365] skipping pod synchronization - [container runtime is down]
Apr 27 20:19:42 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:42.878179    2573 event.go:202] Unable to write event: 'Post http://localhost:8080/api/v1/namespaces/default/events: dial tcp 127.0.0.1:8080: connection refused' (may retry after sleeping)
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:43.429794    2573 factory.go:233] Registering Docker factory
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:43.431034    2573 factory.go:97] Registering Raw factory
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:43.686123    2573 manager.go:1003] Started watching for new ooms in manager
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:43.686835    2573 oomparser.go:182] oomparser using systemd
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:43.687422    2573 manager.go:256] Starting recovery of all containers
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:43.785227105Z" level=error msg="Handler for GET /containers/aff9310361d56f487cf47f1084f9150f4af061db12848c74c5b85bfa1105f450/json returned error: No such container: aff9310361d56f487cf47f1084f9150f4af061db12848c74c5b85bfa1105f450"
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:43.804567    2573 container.go:341] Failed to create summary reader for "/system.slice/system-systemd\\x2dfsck.slice": none of the resources are being tracked.
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:43.805054    2573 container.go:341] Failed to create summary reader for "/system.slice/systemd-fsck-root.service": none of the resources are being tracked.
Apr 27 20:19:43 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: I0427 20:19:43.806507    2573 manager.go:261] Recovery completed
Apr 27 20:19:45 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:45.606173    2573 event.go:202] Unable to write event: 'Post http://localhost:8080/api/v1/namespaces/default/events: dial tcp 127.0.0.1:8080: connection refused' (may retry after sleeping)
Apr 27 20:19:47 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:47.833678    2573 kubelet.go:2307] error getting node: node 'ip-10-0-0-50.ec2.internal' is not in cache
Apr 27 20:19:47 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:47.836666    2573 manager.go:408] Failed to update status for pod "_()": Get http://localhost:8080/api/v1/namespaces/kube-system/pods/kube-apiserver-ip-10-0-0-50.ec2.internal: dial tcp 127.0.0.1:8080: connection refused
Apr 27 20:19:47 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:47.836978    2573 kubelet.go:1764] Failed creating a mirror pod for "kube-apiserver-ip-10-0-0-50.ec2.internal_kube-system(3deb653ddec4807a76b4e5a0f98e1d2c)": Post http://localhost:8080/api/v1/namespaces/kube-system/pods: dial tcp 127.0.0.1:8080: connection refused
Apr 27 20:19:47 ip-10-0-0-50.ec2.internal systemd[1]: Started docker container 5352f0e234b25c22f35cc71d5e850823d8b2d35b61d710071b57f88bec612852.
Apr 27 20:19:47 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:47.986546581Z" level=warning msg="signal: killed"
Apr 27 20:19:48 ip-10-0-0-50.ec2.internal systemd[1]: Stopped docker container 5352f0e234b25c22f35cc71d5e850823d8b2d35b61d710071b57f88bec612852.
Apr 27 20:19:48 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:48.538229381Z" level=error msg="error locating sandbox id c6fc468e9c0788ca59e441a223abadf75b3eb331ebf2a9d7cfa093a2ff37ee6e: sandbox c6fc468e9c0788ca59e441a223abadf75b3eb331ebf2a9d7cfa093a2ff37ee6e not found"
Apr 27 20:19:48 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:48.538277951Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/5352f0e234b25c22f35cc71d5e850823d8b2d35b61d710071b57f88bec612852/shm: invalid argument"
Apr 27 20:19:48 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:48.538297139Z" level=error msg="Error unmounting container 5352f0e234b25c22f35cc71d5e850823d8b2d35b61d710071b57f88bec612852: not mounted"
Apr 27 20:19:48 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:48.538421794Z" level=error msg="Handler for POST /containers/5352f0e234b25c22f35cc71d5e850823d8b2d35b61d710071b57f88bec612852/start returned error: Container command not found or does not exist."
Apr 27 20:19:48 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:48.540383    2573 container.go:341] Failed to create summary reader for "/system.slice/docker-5352f0e234b25c22f35cc71d5e850823d8b2d35b61d710071b57f88bec612852.scope": none of the resources are being tracked.
Apr 27 20:19:48 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:48.548401    2573 manager.go:1858] Failed to create pod infra container: RunContainerError; Skipping pod "kube-apiserver-ip-10-0-0-50.ec2.internal_kube-system(3deb653ddec4807a76b4e5a0f98e1d2c)"
Apr 27 20:19:48 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:48.548798    2573 pod_workers.go:138] Error syncing pod 3deb653ddec4807a76b4e5a0f98e1d2c, skipping: failed to "StartContainer" for "POD" with RunContainerError: "runContainer: API error (500): Container command not found or does not exist.\n"
Apr 27 20:19:49 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:49.823192    2573 kubelet.go:1764] Failed creating a mirror pod for "kube-apiserver-ip-10-0-0-50.ec2.internal_kube-system(3deb653ddec4807a76b4e5a0f98e1d2c)": Post http://localhost:8080/api/v1/namespaces/kube-system/pods: dial tcp 127.0.0.1:8080: connection refused
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal systemd[1]: Started docker container 8a76ee16e463f94836c727a93d47fc327141b5bd6aad724e5cfbb4c9308b7c12.
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:50.526628330Z" level=warning msg="signal: killed"
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal systemd[1]: Stopped docker container 8a76ee16e463f94836c727a93d47fc327141b5bd6aad724e5cfbb4c9308b7c12.
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:50.549352470Z" level=error msg="error locating sandbox id 048aefc41896114f62c0850fc6fb92e930caa59a2c9cf3d34f8345ae96ae9aab: sandbox 048aefc41896114f62c0850fc6fb92e930caa59a2c9cf3d34f8345ae96ae9aab not found"
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:50.549399376Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/8a76ee16e463f94836c727a93d47fc327141b5bd6aad724e5cfbb4c9308b7c12/shm: invalid argument"
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:50.549417010Z" level=error msg="Error unmounting container 8a76ee16e463f94836c727a93d47fc327141b5bd6aad724e5cfbb4c9308b7c12: not mounted"
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:19:50.549527798Z" level=error msg="Handler for POST /containers/8a76ee16e463f94836c727a93d47fc327141b5bd6aad724e5cfbb4c9308b7c12/start returned error: Container command not found or does not exist."
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:50.563684    2573 container.go:341] Failed to create summary reader for "/system.slice/docker-8a76ee16e463f94836c727a93d47fc327141b5bd6aad724e5cfbb4c9308b7c12.scope": none of the resources are being tracked.
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:50.564008    2573 manager.go:1858] Failed to create pod infra container: RunContainerError; Skipping pod "kube-apiserver-ip-10-0-0-50.ec2.internal_kube-system(3deb653ddec4807a76b4e5a0f98e1d2c)"
Apr 27 20:19:50 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:50.564040    2573 pod_workers.go:138] Error syncing pod 3deb653ddec4807a76b4e5a0f98e1d2c, skipping: failed to "StartContainer" for "POD" with RunContainerError: "runContainer: API error (500): Container command not found or does not exist.\n"
Apr 27 20:19:52 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:19:52.833744    2573 manager.go:408] Failed to update status for pod "_()": Get http://localhost:8080/api/v1/namespaces/kube-system/pods/kube-apiserver-ip-10-0-0-50.ec2.internal: dial tcp 127.0.0.1:8080: connection refused
Apr 27 20:19:55 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:19:55.607444    2573 event.go:202] Unable to write event: 'Post http://localhost:8080/api/v1/namespaces/default/events: dial tcp 127.0.0.1:8080: connection refused' (may retry after sleeping)
Apr 27 20:20:02 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:20:02.833677    2573 manager.go:408] Failed to update status for pod "_()": Get http://localhost:8080/api/v1/namespaces/kube-system/pods/kube-apiserver-ip-10-0-0-50.ec2.internal: dial tcp 127.0.0.1:8080: connection refused
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:20:03.834304    2573 kubelet.go:1764] Failed creating a mirror pod for "kube-apiserver-ip-10-0-0-50.ec2.internal_kube-system(3deb653ddec4807a76b4e5a0f98e1d2c)": Post http://localhost:8080/api/v1/namespaces/kube-system/pods: dial tcp 127.0.0.1:8080: connection refused
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal systemd[1]: Started docker container 26f54ef26bf52607f47a3e7561584d638d43b5fa5065824f3b5a30724705b720.
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:03.937493332Z" level=warning msg="signal: killed"
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal systemd[1]: Stopped docker container 26f54ef26bf52607f47a3e7561584d638d43b5fa5065824f3b5a30724705b720.
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:03.982453061Z" level=error msg="error locating sandbox id 46ec4415eaac1cac66fe186c8d7e9bac718773ae6201c62906b82fa16b822a4c: sandbox 46ec4415eaac1cac66fe186c8d7e9bac718773ae6201c62906b82fa16b822a4c not found"
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:03.982856777Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/26f54ef26bf52607f47a3e7561584d638d43b5fa5065824f3b5a30724705b720/shm: invalid argument"
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:03.983080445Z" level=error msg="Error unmounting container 26f54ef26bf52607f47a3e7561584d638d43b5fa5065824f3b5a30724705b720: not mounted"
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:03.983426794Z" level=error msg="Handler for POST /containers/26f54ef26bf52607f47a3e7561584d638d43b5fa5065824f3b5a30724705b720/start returned error: Container command not found or does not exist."
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:20:03.984023    2573 manager.go:1858] Failed to create pod infra container: RunContainerError; Skipping pod "kube-apiserver-ip-10-0-0-50.ec2.internal_kube-system(3deb653ddec4807a76b4e5a0f98e1d2c)"
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:20:03.984068    2573 pod_workers.go:138] Error syncing pod 3deb653ddec4807a76b4e5a0f98e1d2c, skipping: failed to "StartContainer" for "POD" with RunContainerError: "runContainer: API error (500): Container command not found or does not exist.\n"
Apr 27 20:20:03 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:20:03.985107    2573 container.go:341] Failed to create summary reader for "/system.slice/docker-26f54ef26bf52607f47a3e7561584d638d43b5fa5065824f3b5a30724705b720.scope": none of the resources are being tracked.
Apr 27 20:20:05 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:20:05.608650    2573 event.go:202] Unable to write event: 'Post http://localhost:8080/api/v1/namespaces/default/events: dial tcp 127.0.0.1:8080: connection refused' (may retry after sleeping)
Apr 27 20:20:12 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:20:12.833578    2573 manager.go:408] Failed to update status for pod "_()": Get http://localhost:8080/api/v1/namespaces/kube-system/pods/kube-apiserver-ip-10-0-0-50.ec2.internal: dial tcp 127.0.0.1:8080: connection refused
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:20:14.834032    2573 kubelet.go:1764] Failed creating a mirror pod for "kube-apiserver-ip-10-0-0-50.ec2.internal_kube-system(3deb653ddec4807a76b4e5a0f98e1d2c)": Post http://localhost:8080/api/v1/namespaces/kube-system/pods: dial tcp 127.0.0.1:8080: connection refused
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal systemd[1]: Started docker container c1bd4ed99fd5177a64cb27f49453dd0e617ea5e65952470e4ec67a74b2ef9216.
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:14.952373077Z" level=warning msg="signal: killed"
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal systemd[1]: Stopped docker container c1bd4ed99fd5177a64cb27f49453dd0e617ea5e65952470e4ec67a74b2ef9216.
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:14.984030014Z" level=error msg="error locating sandbox id f3a883137aa0f2727af694a1b5a21f69a68024db5bfd0f5cd2c2c3a28161146e: sandbox f3a883137aa0f2727af694a1b5a21f69a68024db5bfd0f5cd2c2c3a28161146e not found"
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:14.984085818Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/c1bd4ed99fd5177a64cb27f49453dd0e617ea5e65952470e4ec67a74b2ef9216/shm: invalid argument"
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:14.984104875Z" level=error msg="Error unmounting container c1bd4ed99fd5177a64cb27f49453dd0e617ea5e65952470e4ec67a74b2ef9216: not mounted"
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal dockerd[1044]: time="2016-04-27T20:20:14.984238013Z" level=error msg="Handler for POST /containers/c1bd4ed99fd5177a64cb27f49453dd0e617ea5e65952470e4ec67a74b2ef9216/start returned error: Container command not found or does not exist."
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:20:14.985165    2573 manager.go:1858] Failed to create pod infra container: RunContainerError; Skipping pod "kube-apiserver-ip-10-0-0-50.ec2.internal_kube-system(3deb653ddec4807a76b4e5a0f98e1d2c)"
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:20:14.985234    2573 pod_workers.go:138] Error syncing pod 3deb653ddec4807a76b4e5a0f98e1d2c, skipping: failed to "StartContainer" for "POD" with RunContainerError: "runContainer: API error (500): Container command not found or does not exist.\n"
Apr 27 20:20:14 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: W0427 20:20:14.986466    2573 container.go:341] Failed to create summary reader for "/system.slice/docker-c1bd4ed99fd5177a64cb27f49453dd0e617ea5e65952470e4ec67a74b2ef9216.scope": none of the resources are being tracked.
Apr 27 20:20:15 ip-10-0-0-50.ec2.internal kubelet-wrapper[2573]: E0427 20:20:15.610000    2573 event.go:202] Unable to write event: 'Post http://localhost:8080/api/v1/namespaces/default/events: dial tcp 127.0.0.1:8080: connection refused' (may retry after sleeping)
```

**Comment by brandonweeks** [comment]
What does `journalctl -u docker` show? You might be running into coreos/bugs#1246.

**Comment by orkhanM** [comment]
@daveey I've been knocking my head on the wall for a few hours with the exact same issue.

I believe this issue is tied to the AMI, I was able to get a cluster up by changing the releaseChannel parameter in cluster.yaml.

**Comment by aaronlevy** [comment]
Based on the log output, you are almost certainly hitting https://github.com/coreos/bugs/issues/1246

**Comment by daveey** [comment]
@orkhanM yes, looks like adding 

`releaseChannel: "beta"`

fixed it.

**Comment by aaronlevy** [comment]
This should be fixed in CoreOS alpha v1032.0.0 - going to close the issue, but please re-open if you're still seeing problems.
